MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,23 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedError: (() -> Void)?
+	var onHideFeedError: (() -> Void)?
 
 	func loadFeed() {
 		onLoadingStateChange?(true)
```

Comment:
To hide an error when loading starts, you can call `onErrorStateChange?(nil)`
```suggestion
		onLoadingStateChange?(true)
		onErrorStateChange?(nil)
```